### DISCUSSION TITLE
socket io with react

### DISCUSSION THREAD

**Discussion by mani-cafs** [socketio/socket.io#4176] [Q&A]
how to remove particular userid  when user disconnected chat

**Comment by darrachequesne** [comment]
Hi! Could you please explain your issue?

**Reply by mani-cafs** [reply]
when login my id give back to backend when logout that userid remove from server

**Comment by darrachequesne** [comment]
If I understand correctly, you want to disconnect all the session of a particular user, is that right? In that case, you can use the `disconnectSockets()` method:

```js
io.on("connection", (socket) => {
  socket.join(someUserId);
});

// and then upon disconnection
io.in(someUserId).disconnectSockets();
```

Reference: https://socket.io/docs/v4/server-api/#serverdisconnectsocketsclose
